.github/workflows/build-push-docker-image.yml

```diff
@@ -62,10 +118,45 @@ jobs:
           # cache-to: type=gha,mode=max
 
       - name: Run Tests
+        if: ${{ inputs.skip_tests != true || needs.get_changed_files.outputs.should_execute_tests == true }}
         run: |
           docker run ${{ inputs.build_tag }} npm run test:js:ci
           docker run ${{ inputs.build_tag }} npm run test:ts:ci
 
```

Comment:
Move notify as separate .yaml and move it to dt-test*.yaml to trigger when hotfix release flow test cases fail